docs/concepts/hexagonal-architecture.md

```
@@ -0,0 +1,109 @@
+        Rules[Business Logic]
+    end
+
+    Infrastructure --> Application
+    Application --> Domain
+```
+
+### 1. Domain Layer (`dvmm-domain`)
+*   **Role:** The "Brain". Contains purely business rules.
+*   **Dependencies:** None. Zero. (No Spring, No SQL, No HTTP).
+*   **Contents:**
+    *   **Aggregates:** (`VmRequestAggregate`) consistency boundaries.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)